source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -23,11 +23,13 @@ Each YAML file has the following keys:
   A phase of the test optionally sends inputs to the client,
   then tests the client's resulting TopologyDescription.
 
-Each phase object has two keys:
+Each phase object has the following keys:
 
+- description: (optional) A textual description of this phase.
```

Comment:
I added the `description` field to make the json files easier to read and make the test phases easier to debug.